wordfugue/blog/models.py

```diff
@@ -28,7 +28,7 @@ def get_published_posts_for_site(self, site):
 
 class BlogPost(TimeStampedModel):
 
-    author = models.ForeignKey(settings.AUTH_USER_MODEL)
+    author = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.DO_NOTHING)
```

Comment:
In Django 2.1, we _need_ to specify the `on_delete` behavior.

One a Django foreign-key relationship, Django needs to know what to do when the DB row on the other end of the foreign-key is deleted. Here, I'm telling Django "do nothing when the user model on the other end is deleted", because I still want the post around, just in case.